lib/project_types/script/commands/create.rb

```diff
@@ -17,6 +17,10 @@ def call(args, _name)
           return @ctx.puts(self.class.help)
         end
 
+        unless name_valid?(form.name)
```

Comment:
I think most other project types put flag/option validation within the form. Checkout the [node create form](https://github.com/Shopify/shopify-app-cli/blob/2539abd571633f74d328c92a9c2160333cf1b3a5/lib/project_types/node/forms/create.rb#L28) for example. Would it make sense to put this in our form?